trino/sqlalchemy/dialect.py

```diff
@@ -66,7 +66,7 @@ def create_connect_args(self, url: URL) -> Tuple[Sequence[Any], Mapping[str, Any
         if url.port:
             kwargs["port"] = url.port
 
-        db_parts = (url.database or "system").split("/")
+        db_parts = (url.database or url.query.get("database") or "system").split("/")
```

Comment:
Pass `database` via query make no sense to me

Reply:
This behaviour was taken from the existing sqlalchemy-trino library https://github.com/dungdm93/sqlalchemy-trino/blob/master/sqlalchemy_trino/dialect.py#L60
Can remove if makes no sense

Reply:
This is because `super(TrinoDialect, self).create_connect_args(url)` spit & merge everything into `kwargs` (and `args`).
`database` is expected in URL path, not URL query.